prov/rxm/src/rxm_cq.c

```diff
@@ -577,7 +579,7 @@ static ssize_t rxm_cq_handle_comp(struct rxm_ep *rxm_ep,
 		assert(comp->flags & (FI_SEND));
 		return rxm_finish_send(tx_entry);
 	case RXM_TX_RMA:
-		assert(comp->flags & (FI_SEND));
+		assert(comp->flags & (FI_WRITE | FI_READ));
```

Comment:
Please make the necessary changes to the prior patch, which sets this.  It's not obvious why FI_READ is needed based on the original code.  Maybe that needs to be in a separate patch?